libraries/botbuilder-dialogs-adaptive/src/skillExtensions.ts

```diff
@@ -25,8 +25,10 @@ export const skillConversationIdFactoryKey = Symbol('SkillConversationIdFactory'
 export class SkillExtensions {
     /**
      * Configures the skill client to use.
+     *
      * @param dialogManager The dialog manager to add skill client to.
      * @param skillClient The skill client to be added.
+     * @returns The dialog manager based on the skill client.
```

Comment:
This return could be changed to something like 
`The existing dialog manager object with the new skill client set into the state.`